### PR TITLE
Changes for Product&Documentation Consistency

### PR DESCRIPTION
**Pull request by aymohammed**
Changed the language of the listed items to match the docs [page](https://www.bigpanda.io/docs/display/BD/POST+alerts). Also, added a few contextual links to that page for customer reference. Please review @erikzaadi.
Thanks!

### CODE REVIEW

**Comment by erikzaadi** [comment]
@aymohammed Looks good, I've put it up at https://stg3-a.bigpanda.io/#/app/integrations/alertsapi/instructions/alertsapi so you can see it live.
@michallevin An extra set of eyes?

File: _integrations/alertsapi.md [hunk 1]
```diff
@@ -22,14 +22,16 @@ The JSON payload can contain all or a subset of the following fields:
...
+|`timestamp` *\*optional*|Time that the alert occured in unix format (UTC timezone). If the time is not specified, the value defaults to the current time.|`"timestamp": 1402302570`|
+|`check` *\*optional*|Secondary object or sub-item that caused the alert (often shown as an incident subtitle in BigPanda).|`"check": "ping"`|
+|`description` *\*optional*|Additional free-form text information about this alert.|`"description": "ping to east-mongodb-1 failed"`|
+|`cluster` *\*optional*|Server cluster or logical host-group from which the alert was sent. This value is used to correlate alerts into high-level incidents.|`"cluster": "east-mongodb"`|
+|Additional attributes *\*optional*|Additional information you want to have available in the BigPanda Incident Dashboard. You can add any number of custom JSON attributes with a string value to the payload.|||
+
+For information on configuring multiple alert parameters or primary and secondary properties, see [Alerts API](https://www.bigpanda.io/docs/display/BD/POST+alerts).
```
Comment (michallevin) [inline _integrations/alertsapi.md]: Looks good.
My only suggestion is to give more information about how to send multiple alerts in this documentation. Note that the 'Sending Multiple Alerts' below is not actually a part of this doc. So maybe add it here in the list of "For information on".... (ex: For information on sending multiple alerts, configuring multiple alert parameters...)

Comment (aymohammed) [inline _integrations/alertsapi.md]: Thanks for the great feedback!
Although, I am a bit confused. I was under the impression they were along
the same line: to send multiple alerts, you configure multiple alert
parameters.
Perhaps it's the language? We can clear it up a bit and maybe change it to
"For information on sending multiple alerts or configuring primary and
secondary properties, see..." What do you think?

Also, I do not have the login for the staging site. Do you?

Thanks!
Alia

**Comment by michallevin** [comment]
@aymohammed 
I actually wasn't sure if 'configuring multiple alert parameters' meant sending multiple alerts, or sending an alert with extra parameters. I like the "To send multiple alerts" :)
And I am not sure what is the best way for you to login to staging-3. @erikzaadi @rasienko ?

**Comment by rasienko** [comment]
@michallevin @aymohammed I don't seem to have access to stg-3 anymore, either. If you figure out how to grant it, can you add access for me too?

**Comment by aymohammed** [comment]
@michallevin @erikzaadi, I've made a sentence change. Let me know if it works!

**Comment by erikzaadi** [comment]
@aymohammed @rasienko Sending credentials via slack (for staging-3)

File: _integrations/alertsapi.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The JSON payload can contain all or a subset of the following fields:
 |-----|-----------|-------|
 |`app_key`|Application key from the first step.|`"app_key": "$STREAM_ID"`|
 |`status`|Status of the alert. One of [ `ok`, `critical`, `warning`, `acknowledged` ].|`"status": "critical"`|
-|`host` */`service`* */`application`* |Main object that caused the alert. Can be the associated host or, if a host isn't relevant, a service or an application. If you want to include more than one of these fields, consider specifying the [primary and secondary properties](https://www.bigpanda.io/docs/display/BD/POST+alerts#POSTalerts-Parameters(PrimaryandSecondaryProperties)).|`"host": "east-mongodb-1"`|
+|`host` */`service`* */`application`* |Main object that caused the alert. Can be the associated host or, if a host isn't relevant, a service or an application. If you want to include more than one of these fields, consider specifying the primary and secondary properties.|`"host": "east-mongodb-1"`|
```
Comment (erikzaadi) [inline _integrations/alertsapi.md]: Was the link removed here on purpose?

Comment (aymohammed) [inline _integrations/alertsapi.md]: Yes, better to have in one place only. They were the same link but there isn't a set way to maintain links here yet, so we want to limit them.

Comment (erikzaadi) [inline _integrations/alertsapi.md]: Kewl

**Comment by rasienko** [comment]
@erikzaadi same here...is this merged/closed?

**Pull request merged**